Print_List_of_Odd_Numbers.py

```diff
@@ -18,6 +18,9 @@ def print_error_messages():
   if(n==0):
     start+=1
     print(start)
-  else:
-    if(n%2!=0):
-    print(start)
+    patch-2
+  elif(n%2!=0):
+    else:
+      if(n%2!=0):
+        master
+      print(start)
```

Comment:
Lack of proper documentation makes it hard to understand the purpose of the program to others.
It would have been better if it was properly documented.